arviz/data/io_pymc3.py

```diff
@@ -1,5 +1,8 @@
 """PyMC3-specific conversion code."""
 import logging
+from typing import Dict, List, Any, Optional, TYPE_CHECKING
+if TYPE_CHECKING:
+    import pymc3 as pm
```

Comment:
won't this break ArviZ for users without PyMC3 installed? Maybe it would be better a try/except?

Reply:
No, this will only break ArviZ for users without PyMC3 installed *who try to use mypy*.  

If you don't use mypy for type checking, then the `if TYPE_CHECKING` block never runs.

Reply:
Is it mypy which needs this import?

Is there a reason for typing stuff (At some point we should add it, but it is not currently implemented?)

Also this needs try except because we might run mypy without pymc3 installed

Reply:
Yes, it is mypy. The type  checking variable will be false when running.

I don't believe this needs try except because 

1. it seems to me to be fine if one cannot run the type checker without PyMC3 installed: we aren't requiring even developers to use mypy. 
2. I'm pretty sure that the error would be in mypy, not in arviz. And I don't think one should try to type check without PyMC3 installed.

I am typing stuff because it helps me to understand the code - it makes it more self documented, and because it helps me avoid introducing errors. It's gradual typing, so does not demand that all the code be annotated. I typically add annotations locally when trying to add a fix or enhancement. No one but me has  to change their coding practices, but I'd appreciate it if my annotations were retained  as long as they don't cause problems.